src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/HealthChecks.feature

```diff
@@ -0,0 +1,20 @@
+﻿@innerApi
+@outerApi
+Feature: HealthChecks
+	In order to check if the api is working
+	As a application monitor
+	I want to be told the status of the api
+
+Scenario Outline: Ping returns Ok 
+	When I ping the Outer Api
+	Then the result should return Ok status
+
+Scenario Outline: HealthCheck is called
+	Given the Apprentice Commitments Inner Api is ready and <InnerStatus>
+	When I call the health endpoint of the Outer Api
+	Then the result should show <OuterStatus>
+
+Examples:
+	| InnerStatus | OuterStatus |
+	| Healthy     | Healthy     |
+	| Unhealthy   | Unhealthy   |
```

Comment:
`InnerStatus` will *always* be the same as `OuterStatus` - I wonder if it is really worth having separate columns.  Maybe a single `Status` column will help express that we are expecting them to be identical?

Reply:
It will be different when we have multiple APIs (Notifications/Commitments etc)